test/unittest/tests/block/test_extent_list_insert_wo_block.cpp

```diff
@@ -33,12 +33,12 @@ TEST_CASE("Extent Lists: block_ext_insert", "[extent_list]")
 
     std::vector<WT_EXT **> stack(WT_SKIP_MAXDEPTH, nullptr);
 
+    /* Common setup. */
+    WT_EXTLIST extlist = {};
+
     SECTION("insert into an empty list has one element")
     {
         BREAK;
-        /* Setup. */
-        WT_EXTLIST extlist = {};
```

Comment:
This change removes the independence of the SECTIONs within the TEST_CASE.
* Are "Common setup" and "Common cleanup" both run for each SECTION, or just once before the first SECTION and once after the last SECTION? If the first, this change is desirable. If the later, this change is not desirable.